src/app/organizations/manage/people.component.ts

```diff
@@ -246,6 +247,30 @@ export class PeopleComponent implements OnInit {
         this.actionPromise = null;
     }
 
+    async bulkReinvite() {
+        if (this.actionPromise != null) {
+            return;
+        }
+
+        const users = this.getCheckedUsers().filter(u => u.status === OrganizationUserStatusType.Invited);
+
+        if (users.length <= 0) {
+            this.toasterService.popAsync('error', this.i18nService.t('errorOccurred'),
+                this.i18nService.t('noSelectedUsersApplicable'));
+            return;
+        }
+
+        const request = new UserBulkReinviteRequest(users.map(user => user.id));
+        this.actionPromise = this.apiService.postManyOrganizationUserReinvite(this.organizationId, request);
```

Comment:
It serves as a nice way to prevent multiple api requests from being sent concurrently. But it doesn't set any loading icon or handles any error state hence why I handle it manually.